### ISSUE TITLE
How to deal with webp images

### ISSUE THREAD

**Issue by GuanacoDevs** [tweepy/tweepy#1391]
I'm trying to post `webp` images but I get the following error:
```
headers, post_data = API._pack_image(filename, 4883, form_field='media', f=f)
raise TweepError('Invalid file type for image: %s' % file_type)
tweepy.error.TweepError: Invalid file type for image: None
```
My code works fine with jpg's and png's but not with webp's
#1298 mentions the use of `imhdr` to handle `webp` as a fallback, but have no idea how to implement it.

**Comment by GuanacoDevs** [comment]
I can see that is already a fix for `3.9.0` in 3733fd6 is there anything I can do to get it to work with `3.8.0` I'm using a `google-cloud-function` to build my Tweets is kinda of a bummer, because I built my converter assuming webp availability.

**Comment by GuanacoDevs** [comment]
Released in 17700c6

**Comment by Harmon758** [comment]
This was indeed resolved with https://github.com/tweepy/tweepy/commit/3733fd673b04b9aa193886d6b8eb9fdaf1718341 as part of v3.9.0 and is a duplicate of #1298.
